README.md

```diff
@@ -1337,8 +1337,9 @@ Alternative that works on all networks.
 ```bash
 curl -s https://api.ipify.org && echo
 ```
-#### Show Summary of Network Information
-This displays all your network interface information, both for IPv4 and IPv6.
+
+#### Show Network Interface Information
+Undocumented flag of the `scutil` command.
```

Comment:
This is a documented flag. From `scutil --help`:

```
or: scutil --nwi
    show network information
```

Reply:
Interesting. It’s not documented in the man file, which I usually consider documentation. Thanks for the heads up.